build-system/test-configs/config.js

```diff
@@ -124,33 +124,26 @@ const lintGlobs = [
   // To ignore a file / directory, add it to .eslintignore.
 ];
 
+/**
+ * This should not include .js files, since those are handled by eslint:
+ *  - required terms: notice/notice
+ *  - forbidden terms; local/forbidden-terms
+ */
 const presubmitGlobs = [
-  '**/*.{css,js,go}',
+  '**/*.{css,go}',
   // This does match dist.3p/current, so we run presubmit checks on the
   // built 3p binary. This is done, so we make sure our special 3p checks
   // run against the entire transitive closure of deps.
   '!{node_modules,build,dist,dist.tools,' +
     'dist.3p/[0-9]*,dist.3p/current,dist.3p/current-min}/**/*.*',
```

Comment:
I'm not sure if this comment is right, since it seems to exclude `dist.3p/current` in the overall negated rule?

Reply:
The comment was added 6 years ago in https://github.com/ampproject/amphtml/commit/4ca3a994202845ed63f6cd059704d1fc893b60c6#diff-4a803636c6dafbf4467932803f5bbca1274b78e77bd8a624539c6d9981f68ce8R90 and things have changed since then. I think it can be removed.